fraud-detection-update-sample/fraud_detection_process_sample.py

```diff
@@ -8,7 +8,7 @@
 
 """Global Variable definition"""
 
-ISOSCAPES_EE_PATH = 'projects/kazob-370920/assets/isoscapes'
+ISOSCAPES_EE_PATH = 'projects/river-sky-386919/assets/isoscapes/d18O_isocape'
```

Comment:
you're setting this to d18O_isocape but I believe there's some code below to read the whole folder and get the path for d18O_isocape and the other isoscapes (e.g. line 69)

Should we remove all of that? Or maybe better, keep this pointing to the folder rather than the file, and put a comment below near line 69 that we're currently only using the oxygen isoscape and using the other ones is pending on isoscape improvements?

Reply:
you're right, this should just be "projects/river-sky-386919/assets/isoscapes"